bingo/chromosomes/chromosome.py

```diff
@@ -8,30 +8,31 @@
 import copy
 from abc import ABCMeta, abstractmethod
 
+from ..local_optimizers.local_opt_interface import LocalOptimizationInterface
```

Comment:
I think we can probably get rid of this separate interface file, since the base class already has dummy versions of the functions.

what do you think?